src/foam/nanos/controller/ApplicationController.js

```diff
@@ -346,6 +348,15 @@ foam.CLASS({
     {
       name: 'styles',
       factory: function() { return {}; }
+    },
+    {
+      class: 'FObjectProperty',
+      name: 'ipAddressInfo',
```

Comment:
If still needed, just make a new NSpec, instead of adding to ApplicationController.